packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -238,6 +238,45 @@ function toConversationMessages(partialRendering: AI.PartiallyRendered[]): Conve
   });
 }
 
+async function loggableMessage(
+  message: ConversationMessage,
+  render: AI.RenderContext['render'],
+  measureCost?: (message: ConversationMessage, render: AI.ComponentContext['render']) => Promise<number>
```

Comment:
Nit: I'd rather have this name and the one in `renderToConversation` match. I don't have a preference as to which one to keep.